config/locales/en/errors.yml

```
@@ -19,6 +19,8 @@ en:
               invalid_date: First published date is not in the correct format
               blank: Enter a first published date
               inclusion: First published date must be between 1/1/1900 and the present
+            html_attachments:
+              missing_contact_reference: "- Contact with ID %{contact_id} embedded in \"%{html_attachment_title}\" doesn't exist"
```

Looking at the screenshot, I'd suggest this should be:

```suggestion
              missing_contact_reference: "- \"%{html_attachment_title}\" - embedded Contact (ID %{contact_id}) doesn't exist"
```

Which would render as:

> HTML attachments - "Test HTML attachment" - embedded Contact (ID 123) doesn't exist